Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/GameModels/Player.cs

```diff
@@ -19,6 +21,18 @@ public class Player : CustomModelErrorMessages<Player> {
 
         public bool IsValid => UserName != null && Location != null;
 
+        public ICollection<Gadget.Gadget> Gadgets { 
+            get => gadgets;
+            set {
+                if(value == null) {
+                    gadgets = new Collection<Gadget.Gadget>();
+                }
+                else gadgets = value;
+            }
```

Comment:
Moet hier achteraan ook niet een `propertyChanged` of zoiets? Dat de view dan ook zo worden geupdate

Reply:
Dat is idd wel handig. In dat geval zou het ook beter een ObservableCollection kunnen zijn.
Maar aan de andere kant, om het in een View te gebruiken zou het alsnog via een ViewModel moeten. Dus misschien dat dat beter in een ViewModel kan komen.